tiledb/sm/array/array.h

```diff
@@ -306,6 +306,11 @@ class Array {
   /*                API                */
   /* ********************************* */
 
+  /** Returns the context resources. */
```

Comment:
This can be accessed through the opened array and I see it's only required in a test.

Reply:
Done.